libs/martech/martech.js

```diff
@@ -71,20 +71,48 @@ const handleAlloyResponse = (response) => {
     .filter(Boolean);
 };
 
+function sendAnalytics(val) {
+  window.alloy('sendEvent', {
+    documentUnloading: true,
+    xdm: {
+      eventType: 'web.webinteraction.linkClicks',
+      web: {
+        webInteraction: {
+          linkClicks: { value: 1 },
```

Comment:
I'm guessing there's no other relevant property we can use other than `linkClicks`?

Reply:
consulted with some analytics experts.  Here was their answer:
We are using fields currently set up to capture data in certain ways. if we were doing something other then this temporary setup we would look to set these values part of other tags, but given this is firing it's own tag it makes more sense for it to be seen as a link click rather then a page view to collect correctly.

Reply:
It would be great to capture the connection type (where possible) through `navigator.connection.effectiveType` and see whether the timeout is hit for just 2g only.